[docs] Update JS examples (#91)
hyperledger/iroha-2-docs: src/guide/javascript.md

Thus, on the plus side, configuration of the client is simple. On the
downside, you have to prepare a lot manually.

A basic client setup looks straightforward:

```ts
import { Client } from '@iroha2/client'

const client = new Client({
  torii: {
    // Both URLs are optional - in case you need only a part of endpoints,
    // e.g. only Telemetry ones
    apiURL: 'http://127.0.0.1:8080',
    telemetryURL: 'http://127.0.0.1:8081',
  },

> })

Reviewer: The main idea is `Client` in general requires `accountId` and `keyPair`, so this example which lacks them is misleading and that's why this has been removed, right @6r1d @0x009922 @appetrosyan ?

Author: To be more specific, `accountId` is always necessary, but `keyPair` is needed for performing transactions and queries.

Reviewer 2: To be even more specific, nothing is necessary. You can pass only `torii.telemetryURL` if you only need to query for Iroha status (with `Client.getStatus()`). This case is not really relevant to the example in the tutorial. In the tutorial we need firstly get the account id and the key pair, and only then we are able to construct the client to proceed to the next chapters. I guess that is the reason why this section is removed from here and defined a bit later, after key pair generation.

Reviewer: Thank you. Now I have the same understanding as you @0x009922

})
```

> A basic client setup requires a Torii configuration and an account ID and
> this allows you to perform the basic operations like a health or status check.

Reviewer 3: ```suggestion
A basic client setup requires a Torii configuration and an account ID.
This allows you to perform basic operations like health or status checks.
```

> As described above, to use transactions or queries you'll need to have a `keyPair` parameter.
> In this code, it is present in the `Client` instance definition:

Reviewer 3: ```suggestion
As described above, to use transactions or queries you'll need to have a `keyPair` parameter
as a part of the `Client` instance definition:
```

```ts
import { Client } from '@iroha2/client'

const client = new Client({
  torii: {

>     // Both URLs are optional - in case you need only a part of endpoints,
>     // e.g. only Telemetry ones

Reviewer 3: ```suggestion
    // Both URLs are optional in case you only need one of them,
    // e.g. only the telemetry endpoints
```

    apiURL: 'http://127.0.0.1:8080',
    telemetryURL: 'http://127.0.0.1:8081',
  },
  accountId: AccountId({

>     // Name for an account
>     name: 'alice',
>     // A domain where account above is located

Reviewer 3: ```suggestion
    // Account name
    name: 'alice',
    // The domain where this account is registered
```

    domain_id: DomainId({
      'name': 'wonderland'
    })
  }),

>   // A key pair, needed for performing transactions or queries

Reviewer 3: maybe
```suggestion
  // A key pair, needed for submitting transactions and queries
```
or just
```suggestion
  // A key pair, needed for transactions and queries
```

```ts
async function ensureDomainExistence(domainName: string) {
  // Query all domains
  const result = await client.request(QueryBox('FindAllDomains', null))
  // Display status

>   console.log('%o', result)

Reviewer 2: ```suggestion
  console.log('%o', result)
  
```

    apiURL: `http://localhost:8080`,
    telemetryURL: `http://localhost:8081`,
  },

>   // Name and domain for the account

Reviewer 3: ```suggestion
  // Account name and the domain where it's registered
```
